src/gui/command.ts

```diff
@@ -112,12 +112,15 @@ export class FindElementsFileCommand {
     }
 
     private static async commandEntry() {
-        const result = await window.showInputBox({
-            placeHolder: l10n.t("Enter Python executable path. `python` value will be used if the field is empty"),
-            prompt: l10n.t("This path is used to locate the visual element descriptors file"),
-        });
-        if (result === undefined) {
-            return;
+        let pythonPath = "";
+        try {
+            pythonPath = await commands.executeCommand(
+                "python.interpreterPath",
+                workspace.workspaceFolders?.map(({ uri }) => uri.fsPath)
+            );
+            pythonPath = pythonPath || workspace.getConfiguration("python").get("defaultInterpreterPath", "python");
```

Comment:
I would put this line outside of the try block so that you can get a python path even if the command has an error (ie if workspace.workspaceFolders is undefined)

Reply:
I am thinking about it too. Thanks alot!